.env.example

```
@@ -53,6 +53,12 @@ XAI_API_KEY=your-xai-api-key-here
 # Cortex API Key
 CORTEX_API_KEY=your-cortex-api-key-here
 
+# Scaleway Credentials (for deployment)
+SCW_ACCESS_KEY=your-access-key
+SCW_SECRET_KEY=your-secret-key
+SCW_DEFAULT_ORGANIZATION_ID=your-org-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SCW_DEFAULT_ORGANIZATION_ID key should go before the SCW_SECRET_KEY key

(UnorderedKey)